ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -29,6 +29,7 @@ export class ProjectNameNotExistsValidator implements Validator {
                 if (result == null) {
                     resolve(null);
                 } else {
+                    // When updating, it's ok to enter the same name.
                     if (this.project != null && result == this.project.id)
                         resolve(null);
                     else
```

Comment:
Meh, adding comments in unrelated PRs doesn't bother me much. We could use all of the helpful comments that we can get. :)